File: Lecture-12/Lecture/Code/Graph/GraphPart1.cpp
template<typename V> @\textbf{Graph<V>::Graph}@() { }
template<typename V> @\textbf{Graph<V>::\~{}Graph}@() { }

template<typename V>
void @\textbf{Graph<V>::addVertex}@(V vertex) {
  _vertices.push_back(vertex);
}

template<typename V>
void @\textbf{Graph<V>::addEdge}@(V from, V to) {
  _edges[from].push_back(to);
}

File: Lecture-12/Code/Graph/GraphPart1.py
class Graph:
	def __init__(self):
		self.vertices = []
		self.edges = []
	
	def addVertice(self, vert):
		self.vertices.append(vert)
	
	def addEdge(self, fromVert, toVert):
		self.edges.append((fromVert, toVert))
	
	...
